x-pack/lib/filters/geoip/database_manager.rb

```diff
@@ -90,10 +96,14 @@ def prepare_metadata
   # update metadata timestamp for those dbs that has no update or a valid update
   # do daily check and clean up
   def execute_download_job
+    success_cnt = 0
+
     begin
       pipeline_id = ThreadContext.get("pipeline.id")
       ThreadContext.put("pipeline.id", nil)
 
+      @metric.namespace([:download]).gauge(:status, :checking)
```

Comment:
I think this assignment could be extracted in a method `update_download_status(:checking)`. In this way the intention is more evident without the read immediately deep into the knowledge of metrics.